.packit.sh

```
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
```

`set -eo pipefail` would be a really, really good idea here: fail early on error, instead of cascading into a much-harder-to-debug one later.